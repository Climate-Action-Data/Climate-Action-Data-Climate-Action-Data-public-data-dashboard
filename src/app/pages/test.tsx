'use client'
import { type NextPage } from "next";

const Test: NextPage = () => {
    return (
        <div>
            <h1>Hello</h1>
            <p>data</p>
        </div>
    )
}

export default Test