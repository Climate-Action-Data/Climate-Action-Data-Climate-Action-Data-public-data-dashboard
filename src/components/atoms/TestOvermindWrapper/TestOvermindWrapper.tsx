import { createOvermind } from 'overmind'
import { Provider } from 'overmind-react'
import { config } from '@/overmind'
import { PropsWithChildren } from 'react'

const overmind = createOvermind(config)

export const TestOvermindWrapper = (props: PropsWithChildren) => {
  return <Provider value={overmind}>{props.children}</Provider>
}
