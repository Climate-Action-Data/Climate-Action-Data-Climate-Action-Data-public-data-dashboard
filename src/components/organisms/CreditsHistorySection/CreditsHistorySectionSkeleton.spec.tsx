import { render } from '@testing-library/react'

import { TestOvermindWrapper } from '@/test/TestOvermindWrapper'
import CreditsHistorySkeleton from '@/components/organisms/CreditsHistorySection/CreditsHistorySectionSkeleton'

test(`renders loading state when creditsHistoryData is undefined`, () => {
  const { container } = render(
    <TestOvermindWrapper>
      <CreditsHistorySkeleton />
    </TestOvermindWrapper>,
  )
  expect(container).toMatchSnapshot()
})
