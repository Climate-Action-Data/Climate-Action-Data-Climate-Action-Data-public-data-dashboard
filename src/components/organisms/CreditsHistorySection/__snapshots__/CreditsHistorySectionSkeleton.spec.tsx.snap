// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loading state when creditsHistoryData is undefined 1`] = `
<div>
  <div
    class="css-9ty7ec"
    id="issued-retired-chart-header"
  >
    <div
      class="css-f5ds6o"
    >
      <div
        class="chakra-stack css-c3wesr"
      >
        <div
          class="chakra-stack css-6uacnb"
        >
          <div
            class="chakra-skeleton css-10x8ef7"
          />
          <p
            class="chakra-text css-0"
          >
            Issued
          </p>
        </div>
        <div
          class="chakra-stack__divider css-2if2sn"
        />
        <div
          class="chakra-stack css-6uacnb"
        >
          <div
            class="chakra-skeleton css-10x8ef7"
          />
          <p
            class="chakra-text css-0"
          >
            Retired
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-1r9qc8o"
    >
      <div
        class="chakra-skeleton css-4e2to2"
      />
    </div>
  </div>
</div>
`;
